.github/labeler.yml

```
@@ -0,0 +1,91 @@
+# Add a Terraform label to changes under the related folders:
+#
+# Folder                      | Labels
+# --------------------------- | ------------------------------------------------
+# terraform                   | Terraform CE
+# terraform-cdk               | None
+# terraform-docs-agents       | Terraform agents
+# terraform-docs-common       | HCP Terraform
+# terraform-enterprise        | Terraform Enterprise
+# terraform-migrate           | Terraform migrate
+# terraform-plugin-framework  | TF Plugin Framework
+# terraform-plugin-log        | None
+# terraform-plugin-mux        | None
+# terraform-plugin-sdk        | None
+# terraform-plugin-testing    | None
+
+Terraform CE:
+- any:
+  - changed-files:
+    - any-glob-to-any-file: [
+        'content/terraform/**'
+      ]
+
+Terraform agents:
+- any:
+  - changed-files:
+    - any-glob-to-any-file: [
+        'content/terraform/**'
+      ]
+
+HCP Terraform:
+- any:
+  - changed-files:
+    - any-glob-to-any-file: [
+        'content/terraform/**'
+      ]
+
+Terraform Enterprise:
+- any:
+  - changed-files:
+    - any-glob-to-any-file: [
+        'content/terraform-enterprise/**'
+      ]
+
+Terraform migrate:
+- any:
+  - changed-files:
+    - any-glob-to-any-file: [
+        'content/terraform-docs-common/**'
```

I think this should be migrate:
```suggestion
        'content/terraform-migrate/**'
```

---

Yeah. I just realized I didn't update the paths after copy/pasting 🤦🏽‍♀️ Fixing that now